.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
```

Comment:
I looked further and it looks like [coveragerc isn't an option because the template is being used instead](https://github.com/OCA/maintainer-quality-tools/blob/master/travis/travis_install_nightly#L69). It would probably be best to just use
```
#pragma: no cover
```

near the top of the python files then 😦